benchmarks/bench_partial_qsort.hpp

```diff
@@ -0,0 +1,76 @@
+#include "bench-qsort-common.h"
+
+template <typename T>
+static void avx512_partial_qsort(benchmark::State& state) {
+    if (!cpu_has_avx512bw()) {
+        state.SkipWithMessage("Requires AVX512 BW ISA");
+    }
+    if ((sizeof(T) == 2) && (!cpu_has_avx512_vbmi2())) {
+        state.SkipWithMessage("Requires AVX512 VBMI2 ISA");
+    }
+    // Perform setup here
+    size_t ARRSIZE = state.range(0);
+    std::vector<T> arr;
+    std::vector<T> arr_bkp;
+
+    /* Initialize elements */
+    arr = get_uniform_rand_array<T>(ARRSIZE);
+    arr_bkp = arr;
+
+    /* Choose random index to sort up until */
+    int k = get_uniform_rand_array<int64_t>(1, ARRSIZE, 1).front();
```

Comment:
Lets modify this benchmark to use k as an argument. We could benchmark with array size fixed to 10000 and various of values of `k = {10, 100, 1000, 5000}`.

Reply:
`avx512_partial_qsort` benchmarks updated to allow `k` to vary.